### PR TITLE
Issue #000 feat: Cassandra trigger deployment changes

### PR DESCRIPTION
**Pull request by PradyumnaNagendra**

### CODE REVIEW

File: ansible/roles/cassandra-db-update/templates/logback.j2 [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                <pattern>%d %msg%n</pattern>
+            </encoder>
+        </appender>
+        <appender name="ASYNC_TRANSACTION_HANDLER_QUESTION_DATA" class="ch.qos.logback.classic.AsyncAppender">
+            <queueSize>1024</queueSize>
+            <discardingThreshold>0</discardingThreshold>
+            <includeCallerData>true</includeCallerData>
+            <appender-ref ref="TRANSACTION_HANDLER_QUESTION_DATA" />
+        </appender>
+        <logger name="org.sunbird.cassandra.triggers.{{ content_keyspace_name }}.question_data" level="INFO" additivity="false">
```
Comment (PradyumnaNagendra) [inline ansible/roles/cassandra-db-update/templates/logback.j2]: @gandham-santhosh : will content_keyspace_name get resolved during copy from xml also? If not, please let me know the correct way to do it.

File: ansible/cassandra-trigger-deploy.yml [hunk 2]
```diff
@@ -8,6 +8,12 @@
         dest: /etc/cassandra/triggers
       with_fileglob:
            - static-files/cassandra.transaction-event-handler-*.jar
+
+    - name: copy logback.xml file to cassandra trigger path
+      template:
+        src: "roles/cassandra-db-update/templates/logback.j2"
```
Comment (gandham-santhosh) [inline ansible/cassandra-trigger-deploy.yml]: Reference to the source file can be just the name of the file. We dont need to specify the full path of the file
src: logback.j2

Comment (PradyumnaNagendra) [reply ansible/cassandra-trigger-deploy.yml]: I added a new role to cassandra-triggers and added logback.j2 file there

File: ansible/roles/cassandra-db-update/templates/logback.j2 (hunk 1 above)
Comment (gandham-santhosh) [reply ansible/roles/cassandra-db-update/templates/logback.j2]: Yes. It will get resolved

File: ansible/cassandra-trigger-deploy.yml (hunk 2 above)
Comment (gandham-santhosh) [reply ansible/cassandra-trigger-deploy.yml]: cassandra-trigger-deploy.yml is the main playbook. Please refer the cassandra-db-update role in this file and mention the template copy  steps in cassandra-db-update/tasks/main.yml

**Pull request merged**